### PR TITLE
Document vars_flags.c

### PR DESCRIPTION
**Pull request by cbt6**

### CODE REVIEW

File: src/vars_flags.c [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+void VarsFlags_Init (VarsFlags * varsFlags)
+{
+    memset(varsFlags, 0, sizeof(VarsFlags));
+}
+
+VarsFlags * SaveData_GetVarsFlags (SaveData * saveData)
+{
+    return (VarsFlags *)SaveData_SaveTable(saveData, SAVE_TABLE_ENTRY_VARS_FLAGS);
```
Comment (lhearachel) [inline src/vars_flags.c:25]: Is this cast necessary?

File: src/vars_flags.c [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+{
+    if (flagID == 0) {
+        return NULL;
+    } else if (flagID < UNKNOWN_FLAGS_START) {
+        GF_ASSERT((flagID / 8) < (NUM_FLAGS / 8));
+        return &varsFlags->flags[flagID / 8];
+    } else {
+        GF_ASSERT(((flagID - UNKNOWN_FLAGS_START) / 8) < (NUM_UNKNOWN_FLAGS / 8));
+        return &(sUnknownFlags[(flagID - UNKNOWN_FLAGS_START) / 8]);
+    }
```
Comment (lhearachel) [inline src/vars_flags.c:60]: Unnecessary parens in this block

File: src/vars_flags.c [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+u16 * VarsFlags_GetVarAddress (VarsFlags * varsFlags, u16 varID)
+{
+    GF_ASSERT((varID - VARS_START) < NUM_VARS);
```
Comment (lhearachel) [inline src/vars_flags.c:65]: Unnecessary parens

File: src/vars_flags.c (hunk 1 above)
Comment (cbt6) [reply src/vars_flags.c]: Nope, just forgot to remove it.

**Review by lhearachel — APPROVED** [review]

**Pull request merged**